library/ziti.c

```diff
@@ -95,14 +95,14 @@ static int parse_getopt(const char *q, const char *opt, char *out, size_t maxout
         // found it
         if (strncasecmp(q, opt, optlen) == 0 && (q[optlen] == '=' || q[optlen] == 0)) {
             const char *val = q + optlen + 1;
-            char *end = strchr(val, '&');
+            char *end = strchr(val, L'&');
```

Comment:
hrm. i was pretty sure i pulled/cleaned first - i'll double check this unexpected change.